sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure Cognitive Services Language LUIS Authoring</AssemblyTitle>
     <Description>Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Authoring API.</Description>
-    <Version>3.8.0-preview</Version>
+    <Version>3.2.0-preview.1</Version>
```

Comment:
Shouldn't this have been `3.1.0-preview.1`?

Reply:
@weshaggard  The last released version is 3.1.0. 
https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring

So according to these guidelines here(https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#incrementing-after-release), I changed it to 3.2.0-preview.1

Reply:
Thanks I was looking at the wrong package. This is the correct version.